src/services/rabbitmq/RabbitMQSubscription.ts

```diff
@@ -0,0 +1,87 @@
+import {inject, injectable} from 'inversify';
+import {getLogger} from '../../logger';
+import RabbitMQ from '../../config/rabbitmq';
+import {ChannelWrapper, connect} from 'amqp-connection-manager';
+import {ConfirmChannel, ConsumeMessage} from 'amqplib';
+import {jsonLogOutput} from '../../utils/json';
+
+@injectable()
+export class RabbitMQSubscription {
+    private static readonly logger = getLogger('RabbitMQSubscription');
+
+    private static channelWrapper: ChannelWrapper;
```

Comment:
Why not make a wrapper that can be injected that gives access to the channel or passes the functions from a channel along?